addons/website/static/tests/tours/snippet_popup_and_scrollbar.js

```diff
@@ -117,8 +132,9 @@ wTourUtils.registerWebsitePreviewTour("snippet_popup_and_scrollbar", {
     {
         content: "Duplicate the Media List snippet",
         trigger: ".o_we_customize_panel we-customizeblock-options:contains('Media List') we-button.oe_snippet_clone:first",
-        in_modal: false,
-        run: "click",
+        run() {
+            this.anchor.click();
+        }
```

Comment:
I am probably missing something obvious but why can't it be `run: "click",`?

Reply:
Otherwise the tour fails. It could be perfect to use it. To be investigated...